### PR TITLE
Stylesheet swapper

### PR DESCRIPTION
**Pull request by mxsnyder**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
 	"version": "0.2.0",
```
Comment (martypowell) [inline .vscode/launch.json]: this should probably be ignored

File: js/utility/regex-tools.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+namespacer('bcpl.utility');
+
+bcpl.utility.regexTools = (() => {
+	/**
+	 * Filters a string down to only characters that match the regex.
+	 *
+	 * @param {string} stringToFilter
+	 * @param {RegExp} filterRegex
+	 */
+	const filter = (stringToFilter, filterRegex) => {
```
Comment (martypowell) [inline js/utility/regex-tools.js]: i think we could name this better (I think), not sure what it would be.

File: .vscode/launch.json (hunk 1 above)
Comment (mxsnyder) [reply .vscode/launch.json]: done

File: js/utility/regex-tools.js (hunk 2 above)
Comment (mxsnyder) [reply js/utility/regex-tools.js]: renamed!

**Pull request merged**